### DISCUSSION TITLE
Where is the maximum distance to walk to a bus/railway stop defined and how can I set it?

### DISCUSSION THREAD

**Discussion by 1Maxnet1** [motis-project/motis#396] [Q&A] [answered]
Is there a maximum time/distance motis assumes people are walking to a bus/railway stop? And if so where is it defined and how can I configure it?

**Comment by felixguendling** [comment, marked as answer]
https://github.com/motis-project/motis/blob/master/protocol/intermodal/IntermodalRoutingRequest.fbs#L53-L72

Each mode has a `max_duration`. Easiest way to see the usage is to change the ranges in the UI and see which JSON requests it sends to the server via your browser's dev tools.

![image](https://github.com/motis-project/motis/assets/1915889/abe31bce-9c39-4b02-aeea-990730c8d725)
